example/index.html

```diff
@@ -81,7 +81,7 @@
              viewer = manifestor({
                  manifest: manifest,
                  container: $('#example-container'),
-                 selectedCanvas: manifest.sequences[0].canvases[10]['@id'],
+            perspective:  'detail',
```

Comment:
This is in order to test the behaviour of the initial load. Feel free to change.